backend/homework/7/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,66 @@
+package org.example;
+
+import org.example.factory.FactoryServiceOne;
+import org.example.factory.FactoryServiceTwo;
+import org.example.inventory.InventoryStore;
+import org.example.speaker.Speaker;
+import org.example.speaker.SpeakerService;
+import org.example.tyre.Tyre;
+import org.example.tyre.TyreService;
+import org.example.vehicle.Vehicle;
+import org.example.vehicle.VehicleService;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+
+import java.util.List;
+import java.util.logging.Logger;
+
+public class Main {
+    private static final Logger logger = Logger.getLogger(Main.class.getName());
+    public static void main(String[] args)
+    {
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(SpeakerService.class, TyreService.class, VehicleService.class, InventoryStore.class, FactoryServiceOne.class, FactoryServiceTwo.class);
+
+        SpeakerService speakerService = context.getBean(SpeakerService.class);
+        TyreService tyreService = context.getBean(TyreService.class);
+
+        VehicleService vehicleService = context.getBean(VehicleService.class);
+
+
+        Speaker speaker1 = speakerService.speaker1();
```

Comment:
Use meaningful and descriptive names for your variables rather than using numerical suffixes.